metrics2/metrics2/pom.xml

```diff
@@ -120,6 +120,12 @@
             <groupId>io.helidon.microprofile.config</groupId>
             <artifactId>helidon-microprofile-config</artifactId>
             <scope>runtime</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
Minor comment: make sure to run `mvn dependency:tree -Dscope=runtime` to ensure that this artifact has not been pulled in transitively by some other dependency.  Getting exclusions right often means playing whack-a-mole.

Reply:
If it somehow gets pulled, it would prevent SE apps (like quickstart-se) to compile under GraalVM. Those are compiling and running under GraalVM now.